spatial-operations/README.md

```diff
@@ -0,0 +1,14 @@
+# Spatial operations
+
+![Spatial Operations App](create-geometries.png)
+
+Demonstrates how to perform geometry-on-geometry spatial operations by using ```GeometryEngine```. Two input ```Polygons``` are added as ```Graphics``` to a ```GraphicsOverlay``` and displayed in a ```MapView```. Using the options menu, different spatial operations can be performed, the result of which is shown as a third ```Graphic```, in red, in the ```MapView```. Additionally, the `union` method is used to create an initial geometry with which to set the ```Viewpoint``` of the ```MapView```.
+
```

Comment:
I think it would be good to mention the general developer pattern of using 2 geometries on `GeometryEngine` methods e.g.: 

```java
// perform intersection
GeometryEngine.intersection(inputPolygon1, inputPolygon2)
// perform union
GeometryEngine.union(inputPolygon1, inputPolygon2)
// ...
```

To show simplicity at a high level.